packages/katello/rubygem-pulp_file_client/rubygem-pulp_file_client.spec

```diff
@@ -0,0 +1,96 @@
+# Generated from pulp_file_client-0.0.1b10.dev.1557779852.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name pulp_file_client
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1b10.dev.1557779852
+Release: 1%{?dist}
+Summary: Pulp 3 API Ruby Gem
+Group: Development/Languages
+License: GPLv2
+URL: https://openapi-generator.tech
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby >= 1.9
+Requires: %{?scl_prefix_ruby}ruby(rubygems) > 1.3.1
+Requires: %{?scl_prefix}rubygem(typhoeus) >= 1.0
+Requires: %{?scl_prefix}rubygem(typhoeus) < 2
+Requires: %{?scl_prefix}rubygem(typhoeus) >= 1.0.1
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1
+Requires: %{?scl_prefix_ruby}rubygem(json) < 3
+Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel > 1.3.1
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
+
+%description
+Pulp3 file plugin api library.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/git_push.sh
```

Comment:
Is this needed? Sounds like an internal helper script.

Reply:
probably not, good catch!